pdsc/ingest.py

```diff
@@ -266,10 +301,20 @@ def ingest_idx(label_file, table_file, configpath, outputdir):
         outputdir,
         '%s%s' % (instrument, SEGMENT_DB_SUFFIX)
     )
-    segments = store_segments(outputfile, metadata, config)
+    if body_radius == MARS_RADIUS_M:
+        segments = store_segments(outputfile, metadata, config)
+    else:
+        segments = store_segments(outputfile, metadata, config, body_radius)
 
     outputfile = os.path.join(
         outputdir,
         '%s%s' % (instrument, SEGMENT_TREE_SUFFIX)
     )
-    store_segment_tree(outputfile, segments)
+    if body_radius == MARS_RADIUS_M:
+        store_segment_tree(outputfile, segments)
+    else:
+        if (segments != []):
+            store_segment_tree(outputfile, segments, body_radius)
+        else:
+            print('No segments to save as a tree')
+
```

Comment:
Can remove the if/else block and just always pass in body_radius=body_radius argument.

Did you come across any instances where there were no segments? It might indicate an error or bug if this ever occurs.

Reply:
Remove else statement for no segments -- I think I got this once when doing a tiny case and that's why I had it, although can't really remember...

Also removed the if/else with the Mars radius. I had these types of things in there in case someone had code that called these functions without the radius (was trying to make them backward compatible), but I think these are all internal so should be ok.

Updated test/test_ingest.py to assume Mars radius passed in:
    mock_store_segment_tree.assert_called_with(
        os.path.join('test_output', 'instrument_name_segment_tree.pkl'),
        'segments', **3396200.0**
    )